Services/CO.CDP.EntityVerification/Api/PponEndpointExtensions.cs

```diff
@@ -9,8 +12,11 @@ public static class PponEndpointExtensions
 {
     public static void UsePponEndpoints(this WebApplication app)
     {
-        app.MapGet("/identifiers/{identifier}", (string _) => Task.FromResult(Results.NotFound()))
-            .Produces<string>(StatusCodes.Status200OK, "application/json")
+        app.MapGet("/identifiers/{identifier}",
+            async (string identifier, IUseCase<LookupIdentifierQuery, IEnumerable<Identifier>> useCase) =>
+                await useCase.Execute(new LookupIdentifierQuery(identifier))
+                    .AndThen(identifier => identifier != null ? Results.Ok(identifier) : Results.NotFound()))
```

Comment:
This is actually "identifiers", right?

Reply:
Ah you mean it should be LookupIdentifier**s**Query